inlong-dataproxy/dataproxy-docker/dataproxy-docker.sh

```diff
@@ -24,10 +24,10 @@ sed -i "s/manager.hosts=.*$/manager.hosts=${MANAGER_OPENAPI_IP}:${MANAGER_OPENAP
 sed -i "s/audit.proxys=.*$/audit.proxys=${AUDIT_PROXY_URL}/g" "${common_conf_file}"
 
 # start
-if [ $MQ_TYPE == "pulsar" ]; then
+if [ "${MQ_TYPE}" = "pulsar" ]; then
```

Comment:
Is it ok to change `==` to `=`?

Reply:
yes, I tested
<img width="795" alt="WeChatWorkScreenshot_a9d6ee64-89a3-45ad-a388-4dc450b93baa" src="https://user-images.githubusercontent.com/100204617/178463890-bc97b25e-8d0e-4cf1-a5d8-96b563f71770.png">